src/main/java/vg/civcraft/mc/citadel/command/Acid.java

```diff
@@ -105,4 +110,17 @@ public boolean execute(CommandSender sender, String[] args) {
 	public List<String> tabComplete(CommandSender sender, String[] args) {
 		return new ArrayList<>();
 	}
+
+	public boolean isChestBeingAcidBlocked (Block block) {
+		if (block.getType() != Material.CHEST || block.getType() != Material.TRAPPED_CHEST) {
+			return false;
+		}
+		Chest chest = (Chest) block.getBlockData();
+		ItemStack[] items =  chest.getBlockInventory().getStorageContents();
+		chest.getBlockInventory().clear();
```

Comment:
Did you test all kinds of edge conditions involving double chests with this?

Reply:
Did some basic testing, single chests alone dropped contents ok
double chest when acid blocking left side dropped nothing but chest 
acid blocking a double chest at once also dropped correct inventory

Reply:
seems to work okay from what ive tested as basic uses of it, is there anything complex you want me to test about it?

Reply:
It should drop half when you acid one half

Reply:
Didn't fill the chest enough on first try, this works as is